core/src/err/mod.rs

```diff
@@ -1149,6 +1149,10 @@ pub enum Error {
 
 	#[error("Found a non-computed value where they are not allowed")]
 	NonComputed,
+
+	/// Represents a failure in timestamp arithmetic related to database internals
+	#[error("Failed to compute: \"{0}\", but one or both operands are too big.")]
```

Comment:
```suggestion
	#[error("Failed to compute: \"{0}\", resulting in an overflow.")]
```

The phrasing "one or both operands are too big" can be misleading:
- Overflow is primarily caused by the result of an operation exceeding the type's limits, not necessarily because the individual operands are too large
- In multiplication, even small operands can produce a large result that causes an overflow

Reply:
What about this?
```suggestion
	#[error("Failed to compute: \"{0}\", as the operation results in an overflow.")]
```